src/test/java/Homework19.java

```diff
@@ -0,0 +1,32 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+    @Test
+    public static void deletePlaylist() throws InterruptedException{
+
+        String email = "demo@class.com";
+        String password = "te$t$tudent";
+
+        enterCredentials(email, password);
+        clickLogin();
+
+        WebElement eighthSong = driver.findElement(By.xpath(".//nav[@class='side side-nav showing']//li[@class='playlist playlist'][8]"));
+        eighthSong.click();
+        WebElement deletePlaylist = driver.findElement(By.cssSelector(".del.btn-delete-playlist"));
+        deletePlaylist.click();
+
+        WebElement confirmationPopup = driver.findElement(By.xpath(".//div[@class='alertify'][1]//p[text()='Delete the playlist \"Test Pro Edited Playlist\"?']"));
+        Assert.assertTrue(confirmationPopup.isDisplayed());
+        WebElement clickOK = driver.findElement(By.xpath(".//div[@class='alertify'][1]//button[@class='ok']"));
+        clickOK.click();
+
+        /* I manually tested and this is as far as I can go. I'm unable to check the confirmation notification
+            because there are two confirmation popups and when I click OK or cancel, it's not possible to select the
+            confirmation notification element to find a selector
+         */
```

Comment:
HINT

`By.cssSelector("div.success.show")`

![image](https://user-images.githubusercontent.com/104761482/235407463-672c5728-57d6-4eb8-ae1b-945c83f12f5f.png)

![image](https://user-images.githubusercontent.com/104761482/235407647-4457ba8f-eba2-4c8f-8f57-bae764eeba43.png)

Browse to the desired page
1.	Open the dev console - F12 on Windows/Linux or option + ⌘ + J on macOS
2.	Select the Sources tab in chrome inspector
3.	In the web browser window, hover over the desired element to initiate the popover
4.	Hit F8 on Windows/Linux (or fn + F8 on macOS or  CTRL+\) while the popover is showing. If you have clicked anywhere on the actual page F8 will do nothing. Your last click needs to be somewhere in the inspector, like the sources tab
5.	Go to the Elements tab in inspector
6.	Find your popover (it will be nested in the trigger element's HTML)
7.	Have fun modifying the CSS
8.	https://www.howtogeek.com/663950/what-your-function-keys-do-in-chrome-devtools/